### ISSUE TITLE
[Solved] How to make Snowball Fight work with KEmulator?

### ISSUE THREAD

**Issue by Darthagnon** [dungnv53/SnowBallFight#1]
Greetings, Sir.

I saw your [YouTube video](https://www.youtube.com/watch?v=rkz_-Vyr4Zw) where you were playing Snowball Fight in KEmulator. I have tried, but I cannot get it to work. I've tried the files from this repo, and files from other places :'(

How did you do it?

**Comment by dungnv53** [comment]
Yeah, It's quite old tools on Windows 7 or 8 but I think it's can still work on windows 10 too.
Please try this jar game file https://drive.google.com/file/d/0B9XwFe7bHCQ0OEJhVm1sNDFCRlk/view?usp=sharing 
KEmulator: https://drive.google.com/file/d/1nBpzpmFSdBaI3MA9yPP6bi6qde8lHYXC/view?usp=sharing
I'm not sure it can work on latest Windows 10, so If you got any problem, please let me know.
Hope this help.

PS: This repo I used for compiling and reverse engineering original game and work only with Eclipse + Plugin for J2ME on Windows 7/8, these setup are very old and outdated and many lib, packages required not available now.

**Comment by Darthagnon** [comment]
Thank you so much! I'm on Win10 v1709; I'll test on Win8 later; unfortunately, it still doesn't work for me, even using the new JAR and emu. The titlescreen displays, the intro song plays, and log writes this:
```
KEmulator Lite v0.9.8 Running...
Get classes from D:\Users\Darren\Downloads\Snowball Fight for Android\Snowball Fight\Dzung v2\KEmulator\KEmulator\Snow_Ball_Fight_suite.jar
Get class SnowBallFight
Get class GameScreen
Custom.jar.getResourceStream: SnowBallFight.png (null)
Launch MIDlet class: SnowBallFight
openRecordStore hero
openRecordStore config
Custom.jar.getResourceStream: /0.png (160)
Custom.jar.getResourceStream: /1.png (163)
Custom.jar.getResourceStream: /2.png (161)
Custom.jar.getResourceStream: /3.png (160)
Custom.jar.getResourceStream: /4.png (162)
Custom.jar.getResourceStream: /5.png (161)
Custom.jar.getResourceStream: /6.png (162)
Custom.jar.getResourceStream: /7.png (162)
Custom.jar.getResourceStream: /8.png (159)
Custom.jar.getResourceStream: /9.png (162)
Custom.jar.getResourceStream: /logo.png (5127)
Custom.jar.getResourceStream: /9.mmf (1248)
```
and then doesn't do anything more :'(

**Comment by dungnv53** [comment]
@Darthagnon I've tried both KEmulator I had shared and both are not work as you mentioned (I extract rar file then run KEmulator.exe).
I did checked on my old PC running Windows 7 Ultimate Service pack 1 and it work well with my already installed version.
I do not keep installer file so I tried zip one version from Program Files/. Not sure it can work.
So please re-try with this KEmulator https://drive.google.com/file/d/1nBpzpmFSdBaI3MA9yPP6bi6qde8lHYXC/view?usp=sharing
Some screenshots 
https://filebin.net/6926c2yypp5v0mr7/KEmu-0.9.8-Lite.png?t=xdrovsme
https://filebin.net/6926c2yypp5v0mr7/SBF-Win7-KEmulator-playing.png?t=xdrovsme
I will try with Windows 10 tomorrow and find what exactly worked KEmulator version (installer or portable).
Hope this help.

Update: I have tried on Windows 10 x64 and it can work too. I will put these file to this source.

**Comment by Darthagnon** [comment]
Tested the new version, and I can confirm: it works! I had to copy the ```/jre``` (and ```/rms``` and ```/language```, for good measure) (I've attached the files I had to copy) from another version of the emulator for it to start (I'm wondering if maybe it has something to do with java version?). Snowball fight doesn't seem to save/progress, but at least it boots and runs! Thank you so much for your help.
[KEmulator_deps.zip](https://github.com/dungnv53/SnowBallFight/files/5306472/KEmulator_deps.zip)
